depmapomics/config_global.py

```diff
@@ -420,56 +424,88 @@
     ],
 }
 
+MAF_COL = "depmap_maf"
+HGNC_MAPPING = "data/new_and_old_hgnc_symbols.txt"
+
 MUTCOL_DEPMAP = [
-    "Hugo_Symbol",
-    "Entrez_Gene_Id",
-    "NCBI_Build",
-    "Chromosome",
-    "Start_position",
-    "End_position",
-    "Strand",
-    "Variant_Classification",
-    "Variant_Type",
-    "Reference_Allele",
-    "Tumor_Allele",
-    "dbSNP_RS",
-    "dbSNP_Val_Status",
-    "Genome_Change",
-    "Annotation_Transcript",
+    "chrom",
+    "pos",
+    "ref",
+    "alt",
+    "af",
+    "ref_count",
+    "alt_count",
+    "gt",
+    "ps",
+    "variant_type",
+    "variant_info",
+    "dna_change",
+    "protein_change",
+    "hugo_symbol",
+    "hgnc_name",
+    "hgnc_family",
+    "transcript",
+    "transcript_exon",
+    "transcript_strand",
+    "uniprot_id",
+    "str",
+    "dbsnp_id",
+    "dbsnp_filter",
+    "issues",
+    "gc_content",
+    "lineage_association",
+    "achilles_top_genes",
+    "cancer_molecular_genetics",
+    "ccle_deleterious",
+    "structural_relation",
+    "cosmic_hotspot",
+    "cosmic_overlapping_mutations",
+    "associated_with",
+    "lof",
+    "driver",
+    "likely_driver",
+    "transcript_likely_lof",
+    "civic_id",
+    "civic_description",
+    "civic_score",
+    "popaf",
+    "likely_gof",
+    "likely_lof",
+    "hess_driver",
+    "hess_signture",
+    "cscape_score",
+    "dann_score",
+    "revel_score",
+    "funseq2_score",
+    "pharmgkb_id",
+    "dida_id",
+    "dida_name",
+    "gwas_disease",
+    "gwas_pmid",
+    "gtex_gene",
     SAMPLEID,
-    "cDNA_Change",
-    "Codon_Change",
-    "Protein_Change",
-    "isDeleterious",
-    "isTCGAhotspot",
-    "TCGAhsCnt",
-    "isCOSMIChotspot",
-    "COSMIChsCnt",
-    "ExAC_AF",
-    "Variant_annotation",
-    "CGA_WES_AC",
 ]
 
 
 ############## FUSION
 
-FUSION_COLNAME = {
-    "FusionName": "fusionName",
-    "JunctionReadCount": "junctionReadCount",
-    "SpanningFragCount": "spanningFragCount",
-    "SpliceType": "spliceType",
-    "LeftGene": "leftGene",
-    "LeftBreakpoint": "leftBreakpoint",
-    "RightGene": "rightGene",
-    "RightBreakpoint": "rightBreakpoint",
-    "LargeAnchorSupport": "largeAnchorSupport",
-    "FFPM": "ffpm",
-    "LeftBreakDinuc": "leftBreakDinuc",
-    "LeftBreakEntropy": "leftBreakEntropy",
-    "RightBreakDinuc": "rightBreakDinuc",
-    "RightBreakEntropy": "rightBreakEntropy",
-    "annots": "annots",
-}
+FUSION_COLNAME = [
+    "FusionName",
+    "JunctionReadCount",
+    "SpanningFragCount",
+    "SpliceType",
+    "LeftGene",
+    "LeftBreakpoint",
+    "RightGene",
+    "RightBreakpoint",
+    "LargeAnchorSupport",
+    "FFPM",
+    "LeftBreakDinuc",
+    "LeftBreakEntropy",
+    "RightBreakDinuc",
+    "RightBreakEntropy",
+    "annots",
```

Comment:
Annots?

Reply:
That column has additional info and has always been there in the release

Reply:
yes but it doesn't follow the naming logic here

Reply:
ask portal team (compile all places that have similar issues)